### PR TITLE
Introduced `make fmt` to pilot.

### PR DESCRIPTION
**Pull request by gyliu513**
**Release note**:

```release-note
None
```

Fixed https://github.com/istio/pilot/issues/1077

/cc @ldemailly

### CODE REVIEW

**Comment by ldemailly** [comment]
can it be hooked to the commit hook ?

**Comment by gyliu513** [comment]
@ldemailly Posted a PR here https://github.com/istio/pilot/pull/1093 to address your comments, please help check.

File: bin/fmt.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# Applies requisite code formatters to the source tree
+
+set -e
+SCRIPTPATH=$( cd "$(dirname "$0")" ; pwd -P )
+source $SCRIPTPATH/use_bazel_go.sh
+
+ROOTDIR=$SCRIPTPATH/..
+cd $ROOTDIR
+
+GO_FILES=$(find adapter cmd model platform proxy test tools/version -type f -name '*.go')
```
Comment (ldemailly) [inline bin/fmt.sh]: why isn't it find .  ?

File: bin/fmt.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+UX=$(uname)
+
+#remove blank lines so gofmt / goimports can do their job
```
Comment (ldemailly) [inline bin/fmt.sh]: can you elaborate what is this about ?

File: bin/fmt.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+for fl in ${GO_FILES}; do
+	if [[ ${UX} == "Darwin" ]];then
+		sed -i '' -e "/^import[[:space:]]*(/,/)/{ /^\s*$/d;}" $fl
+	else
+		sed -i -e "/^import[[:space:]]*(/,/)/{ /^\s*$/d;}" $fl
+	fi
+done
+gofmt -s -w ${GO_FILES}
+goimports -w -local istio.io ${GO_FILES}
```
Comment (ldemailly) [inline bin/fmt.sh]: isn't goimports a super set of gofmt ? (http://goinbigdata.com/goimports-vs-gofmt/)

**Pull request merged**